Content.Client/_DV/Curation/UI/Cwoink/CwoinkWindow.xaml.cs

```diff
@@ -0,0 +1,39 @@
+using Robust.Client.AutoGenerated;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+
+namespace Content.Client._DV.Curation.UI.Cwoink;
+
+/// <summary>
+/// This window connects to a CwoinkSystem channel. CwoinkSystem manages the rest.
+/// </summary>
+[GenerateTypedNameReferences]
+public sealed partial class CwoinkWindow : DefaultWindow
+{
+    public CwoinkWindow()
+    {
+        RobustXamlLoader.Load(this);
+
+        Cwoink.ChannelSelector.OnSelectionChanged += sel =>
+        {
+            if (sel is null)
+            {
+                Title = Loc.GetString("cwoink-title-none-selected");
+                return;
+            }
+
+            Title = $"{sel.CharacterName} / {sel.Username} | {Loc.GetString("generic-playtime-title")}: ";
+
+            Title = $"{sel.CharacterName} / {sel.Username} | {Loc.GetString("generic-playtime-title")}: "
+                + sel.OverallPlaytime != null
+                    ? sel.PlaytimeString
+                    : Loc.GetString("generic-unknown-title");
```

Comment:
The + binds before the `!=` so you need parens, that one's on me :⁾
```suggestion
            Title = $"{sel.CharacterName} / {sel.Username} | {Loc.GetString("generic-playtime-title")}: "
                + (sel.OverallPlaytime != null
                    ? sel.PlaytimeString
                    : Loc.GetString("generic-unknown-title"));
```